ui/src/integrations/ethereum/ethereum.ts

```diff
@@ -15,7 +15,7 @@ const alchemy = new Alchemy(config);
 
 export const Ethereum: Ethereum.Core = {
   //TODO remove

```

Comment:
Do we still need to remove this?

Reply:
Not sure but I think so. @csosamorales

Reply:
@jsonsivar no, gonna remove that comment